scripts/west_commands/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.mypy]
+ignore_missing_imports = true
+mypy_path = "."
+
+[tool.pytest.ini_options]
+pythonpath = ["."]
```

Comment:
Please add a link to the relevant documentation because this is surprisingly complex and cryptic:
https://docs.pytest.org/en/7.1.x/explanation/pythonpath.html